src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,33 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private double weight;
+    private double bladeWidth;
+    private double height;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, double weight, double bladeWidth, double height) {
+        this.setName(name);
+        this.setColor(color);
```

Comment:
it's better to create constructor in Machine and here change to super(name, color);

Reply:
fixed this